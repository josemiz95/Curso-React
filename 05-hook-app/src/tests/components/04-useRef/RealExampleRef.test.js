import { shallow } from "enzyme"
import { RealExampleRef } from "../../../components/04-useRef/RealExampleRef"


describe('Test for <RealExampleRef />', () => { 
	const wrapper = shallow(<RealExampleRef />);

	test('Should show correctly', () => { 
		expect(wrapper).toMatchSnapshot();
		expect(wrapper.find('MultipleCustomHooks').exists()).toBe(false);
	});

	test('should show the component <MultipleCustomHooks />', () => {
		wrapper.find('button').simulate('click');

		expect(wrapper.find('MultipleCustomHooks').exists()).toBe(true);
	});
});