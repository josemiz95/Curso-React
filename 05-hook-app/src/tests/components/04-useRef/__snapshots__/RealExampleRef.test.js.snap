// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for <RealExampleRef /> Should show correctly 1`] = `
<div>
  <h1>
    RealExampleRef
  </h1>
  <hr />
  <button
    className="btn btn-primary mt-5"
    onClick={[Function]}
  >
    show/hide
  </button>
</div>
`;
